overrides.go

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor:
+// - Brandon Myers bmyers@mozilla.com
+
+package main
+
+import (
+  "fmt"
+  "os"
+  "bufio"
+  "strings"
+  "strconv"
+)
+
+type Override struct {
+  CIDR string
+  City string
+  Country string
+  Latitude float64
+  Longitude float64
+}
+
+
+func readOverrides(path string) (overrides []Override){
+  file, err := os.Open(path)
+  defer file.Close()
+  if err != nil {
+    fmt.Printf("error opening file: %v\n",err)
+    return overrides
+  }
+
+  reader := bufio.NewReader(file)
+
+  var line string
+  for {
+    line, err = reader.ReadString('\n')
+    line_contents := strings.TrimSpace(line)
+    if strings.HasPrefix(line_contents, "#"){
+      // The line is a comment, so we'll skip it
+      continue
+    }
+    if line_contents == ""{
+      break
+    }
+
+    elements := strings.Split(line_contents, ",")
+    cidr := elements[0]
+    city := elements[1]
+    country := elements[2]
+    latitude, lat_err := strconv.ParseFloat(elements[3], 64)
+    if lat_err != nil {
+      fmt.Printf("Error converting latitude: %v\n",err)
+      continue
+    }
+    longitude, long_err := strconv.ParseFloat(elements[4], 64)
+    if long_err != nil {
+      fmt.Printf("Error converting longitude: %v\n",err)
+      continue
+    }
+
+    overrides = append(overrides, Override{cidr, city, country, latitude, longitude})
+
+    if err != nil {
```

Comment:
I'm not sure what this err check is for here, is this for the call to ReadString? If so, should this be up near that function call since all the code above here would be operating on undefined data if ReadString does return an error?

Reply:
I think it was originally due to how we're parsing the config file, but I was able to simply remove it when switching over to bufio.NewScanner